app/src/main/java/org/mozilla/fenix/IntentReceiverActivity.kt

```diff
@@ -70,6 +71,25 @@ class IntentReceiverActivity : Activity() {
         launch(intent, intentProcessorType)
     }
 
+    // Sanitizes the intent.  If the intent cannot be unparcelled, all extras is removed.
+    // https://developer.android.com/guide/components/activities/parcelables-and-bundles
+    @VisibleForTesting
+    @Suppress("TooGenericExceptionCaught")
+    internal fun Intent.sanitize(): Intent {
```

Comment:
This looks good, but we wanted to add this function to A-C so we can use it in both Fenix and Focus. :)

Reply:
Sure, I'll put it in A-C